content/en/continuous_integration/intelligent_test_runner/_index.md

```diff
@@ -10,187 +10,71 @@ further_reading:
-
-### .NET
-
-To enable Intelligent Test Runner, the version of the `dd-trace` tool must be >= 2.22.0 (execute `dd-trace --version` to get the version of the tool) and the following environment variables must be set:
-
-`DD_CIVISIBILITY_AGENTLESS_ENABLED`
-: Enables or disables Agentless mode.<br/>
-**Default**: `false`<br/>
-
-`DD_API_KEY` (Required if Agentless is enabled)
-: The [Datadog API key][2] used to upload the test results.<br/>
-**Default**: `(empty)`
-
-`DD_APPLICATION_KEY` (Required)
-: The [Datadog Application key][3] used to query the tests to be skipped.<br/>
-**Default**: `(empty)`
-
-`DD_SITE` (Required)
-: The [Datadog site][4] to upload results to.<br/>
-**Default**: `datadoghq.com`<br/>
-**Selected site**: {{< region-param key="dd_site" code="true" >}}
-
-After setting these environment variables, run your tests as you normally do by using [dotnet test][6] or [VSTest.Console.exe][7]:
-
-{{< tabs >}}
-
-{{% tab "dotnet test" %}}
-
-
-{{< code-block lang="shell" >}}
-dd-trace ci run --dd-service=my-dotnet-app --dd-env=ci -- dotnet test
-{{< /code-block >}}
-
-{{% /tab %}}
-
-{{% tab "VSTest.Console" %}}
-
-
-{{< code-block lang="shell" >}}
-dd-trace ci run --dd-service=my-dotnet-app --dd-env=ci -- VSTest.Console.exe {test_assembly}.dll
-{{< /code-block >}}
-
-{{% /tab %}}
-
-{{< /tabs >}}
-
-#### UI activation
-
-In addition to setting the environment variables, you or a user in your organization with admin permissions must activate the Intelligent Test Runner on the [Test Service Settings][5] page.
-
-### Swift
-
-To enable Intelligent Test Runner, the version of the `dd-sdk-swift` framework must be >= 2.2.0. The `Code Coverage` option must also be enabled in the test settings of your scheme or test plan, or  `--enable-code-coverage` must be added to your `swift test` command (if using a SPM target).
+## Configuration
 
-The following environment variables must also be set:
+Once you have setup your Datadog Library for Intelligent Test Runner, you can configure it from the [Test Service Settings][2] page.
 
-`DD_TEST_RUNNER`
-: Enables or disables the instrumentation of tests. Set this value to `$(DD_TEST_RUNNER)` so you can enable and disable test instrumentation with a environment variable defined outside of the test process (for example, in the CI build).<br/>
-**Default**: `false`<br/>
-**Recommended**: `$(DD_TEST_RUNNER)`
+{{< img src="continuous_integration/itr_overview.png" alt="Intelligent test runner enabled in test service settings in the CI section of Datadog." style="width:80%;">}}
 
-`DD_API_KEY` (Required)
-: The [Datadog API key][2] used to upload the test results.<br/>
-**Default**: `(empty)`
+Due to the limitations described above, the default branch of your repository is automatically excluded from having Intelligent Test Runner enabled. This is the recommended configuration by Datadog to ensure that all of your tests are run in the default branch.
```

Comment:
the last phrase does not explain much, does it? 
Maybe 
```suggestion
Due to the limitations described above, the default branch of your repository is automatically excluded from having Intelligent Test Runner enabled. This is the recommended configuration by Datadog to minimize the risks of false positives. 
```

Reply:
Discussed offline and rephrased.